MUXSDKStats/MUXSDKStats/MUXSDKPlayerBinding.m

```diff
@@ -22,6 +22,8 @@
 // Number of seconds the playhead has to move from the last known playhead position when
 // restarting play to consider the transition to play a seek. (500ms)
 float MUXSDKMaxSecsSeekPlayheadShift = 0.5f;
+// Default to allowing automaticVideoChange to take place
+bool _automaticVideoChange = true;
```

Comment:
I think you want this to be a private instance variable on the player binding? https://github.com/muxinc/mux-stats-sdk-avplayer/blob/dc7a70d5569184223d5c45f76091addd641e2421/MUXSDKStats/MUXSDKStats/MUXSDKPlayerBinding.h#L66

Reply:
I tried doing this, but couldn't seem to get it to default to being `true` - maybe i'm missing something obvious?

Reply:
I think once you make it an instance variable, you have to set it to true in the initializer: https://github.com/muxinc/mux-stats-sdk-avplayer/blob/1fe8a2808dc8d502483253fcbbeaa2e6bf3b309f/MUXSDKStats/MUXSDKStats/MUXSDKPlayerBinding.m#L43